python-frontend/src/main/java/org/sonar/python/semantic/SymbolTableBuilder.java

```diff
@@ -273,11 +269,9 @@ public void visitFileInput(FileInput tree) {
       createScope(tree, null);
       enterScope(tree);
       moduleScope = currentScope();
-      if (!SymbolUtils.isTypeShedFile(pythonFile) || !BASE_MODULES.contains(pythonFile.fileName())) {
-        Map<String, Symbol> typeShedSymbols = TypeShed.builtinSymbols();
-        for (String name : BuiltinSymbols.all()) {
-          currentScope().createBuiltinSymbol(name, typeShedSymbols);
-        }
+      Map<String, Symbol> typeShedSymbols = TypeShed.builtinSymbols();
+      for (String name : BuiltinSymbols.all()) {
```

Comment:
probably we don't need anymore to go through `BuiltinSymbols.all()` , however let's do that in a separate PR :)